libs/design-system/package.json

```diff
@@ -42,7 +42,7 @@
     "axios": "^1.9.0",
     "react-helmet-async": "^1.3.0",
     "react-hook-form": "7.43.9",
-    "react-icons": "^5.0.1",
+    "react-icons": "5.0.1",
```

Comment:
react-icons are used in dashboard with different version, and for some reason in the `web` it was picked with `5.3.0` which had slightly different icon names, but I didn't change that, instead updated the version to constant. This was the reason of `web` build failing.